crates/ruma-common/src/identifiers/signatures.rs

```diff
@@ -32,9 +32,9 @@ pub type EntitySignatures<K> = BTreeMap<OwnedSigningKeyId<K>, String>;
     transparent,
     bound(serialize = "E: Serialize", deserialize = "E: serde::de::DeserializeOwned")
 )]
-pub struct Signatures<E: Ord, K: KeyName + ?Sized>(BTreeMap<E, EntitySignatures<K>>);
+pub struct Signatures<E: Ord + Clone, K: KeyName + ?Sized>(BTreeMap<E, EntitySignatures<K>>);
```

Comment:
This, as well as the `Clone` bounds on the existing impls, is unnecessary. Please revert that part.